### ISSUE TITLE
Uniqueness across workers

### ISSUE THREAD

**Issue by mensfeld** [mhenrixon/sidekiq-unique-jobs#210]
Is there an option to realize uniquness across workers without hacking the ```digestable_hash``` method? I know there's an option to ensure uniqueness across queues but what about workers?

**Comment by mensfeld** [comment]
quick hack to fix that issue:
```ruby
module SidekiqUniqueJobs
  class UniqueArgs
    def digestable_hash
      hash = @item.slice(CLASS_KEY, QUEUE_KEY, UNIQUE_ARGS_KEY)
      hash.delete(QUEUE_KEY) if unique_on_all_queues?
      hash.delete(CLASS_KEY)
      hash
    end
  end
end
```

**Comment by mhenrixon** [comment]
Never thought of that. What is the use case more specifically?

**Comment by mensfeld** [comment]
@mhenrixon  I have a bunch of workers (different) that operate on the same resource and I would love to be able to ensure that only one is working on it at the same time. Situation in rare but can happen.

**Comment by mhenrixon** [comment]
Closed by aabcd28dd5aa4ed11579803d8fa10c4e61c0e63b

**Comment by mensfeld** [comment]
@mhenrixon thank you. You've made my day with that one!
